### DISCUSSION TITLE
求助大佬们 用外网穿透后提示这个错误，内网直接访问就没事

### DISCUSSION THREAD

**Discussion by weituw** [AlistGo/alist#6480] [Q&A]
求职大佬们怀疑是UI的问题，有什么解决的办法吗
System error: Error: [Hope UI]: useColorMode must be used within a HopeProvider
![image](https://github.com/alist-org/alist/assets/89982975/209228fa-5521-4922-85bb-86f1ee8da960)
![image](https://github.com/alist-org/alist/assets/89982975/d96ab21d-7afa-4141-b71d-5ed026dc86f7)

**Comment by weasonn** [comment]
可以参考这条
https://github.com/alist-org/alist/discussions/5539
